### PR TITLE
Move ERT-API project from docker jobs to ECS template

### PR DESCRIPTION
**Pull request by dmkov**
Please review changes in jenkins job for ERT-API project. 

I did not update ECS configuration for the project (https://github.com/CruGlobal/ecs_config/blob/master/ecs/ert-api/aws.json) since it was set for the previous docker image but environment variables file should be regenerated there because of migration "database connection url" -> "database host, database port, db name".

I also was not able to reach 'health-check-path' (/eventhub-api/index.html) mentioned in the configuration (https://screencast.com/t/bNHhv2nw). If it is not something linked to the deploy process we can specify '/docs/' (quick but not really good solution) or I can add 'green.html' file to the project.

### CODE REVIEW

**Comment by mattdrees** [comment]
@hlbraddock one consequence of this change would be the auto-deployment policy. Right now, ERT doesn't auto-deploy when merges to master are made during auto-deploy hours (7am-1pm eastern IIRC). This PR would enable auto-deploy. I think this is a healthy change, since it's the policy being followed by most of our apps and since it has not (as far as I know) resulted in unintended deployments or other problems. Sound OK to you?

**Comment by hlbraddock** [comment]
Very well. I prefer having the flexibility to deploy separate from merge, but as long as I'm aware of the change, I can adapt. Thanks for letting me know.

**Comment by mattdrees** [comment]
I'll return to this next wednesday when i'm back in town

**Comment by hlbraddock** [comment]
@mattdrees Modernized https://github.com/CruGlobal/jenkins-jobs/pull/79/files#diff-4f0945b58804468b37caf17e9e3e2a569d8c9a0078be110100686f3308391711R327 (from 'generic-template').

**Comment by mattdrees** [comment]
thanks Lee

**Comment by hlbraddock** [comment]
I'll merge after we merge https://github.com/CruGlobal/ert-api/pull/907.

**Review by hlbraddock — APPROVED** [review]

**Comment by hlbraddock** [comment]
I'm going to merge this PR prior to merge of https://github.com/CruGlobal/ert-api/pull/908 and https://github.com/CruGlobal/ert-api/pull/907 in order that we might test out the jenkins build, deploy, and QA in staging first.

**Pull request merged**
